### PR TITLE
update TUTORIAL.md

### PR DESCRIPTION
**Pull request by alhamilton1111**
Recently went through the set up process and noticed some missing/updated information that would be useful for beginners. Check it out and let me know what you think. Thanks!

### CODE REVIEW

File: TUTORIAL.md [hunk 1]
```diff
@@ -22,15 +22,19 @@ This tutorial will be easiest if you have some knowledge of how to use Linux com
...
+A. Visit the [Spotify developer portal](https://developer.spotify.
+com/dashboard/). If you already have a Spotify account, click "Log in" and enter your username and password. Otherwise, click "Sign up" and follow the steps to create an account. After you've signed in or signed up, you should be redirected to your developer dashboard. You do not need to have a premium Spotify account, a free account will work.
 
 B. Click the "Create an App" button. Enter any name and description you'd like for your new app. Accept the terms of service and click "Create."
 
-C. In your new app's Overview screen, click the "Edit Settings" button and scroll down to "Redirect URIs." Add "http://localhost:1234" (or any other port number of your choosing). Hit the "Save" button at the bottom of the Settings panel to return to you App Overview screen.
+C. In your new app's Overview screen, click the "Edit Settings" button and scroll down to "Redirect URIs." Add "[https://localhost:1234/callback/]" (or any other port number of your choosing). Hit the "Save" button at the bottom of the Settings panel to return to you App Overview screen.
```
Comment (stephanebruckert) [inline TUTORIAL.md:34]: No need for https

https -> http

File: setup.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
 setup(
     name='spotipy',
-    version='2.21.0',
+    version='2.23.0',
```
Comment (stephanebruckert) [inline setup.py:21]: Why 2.23.0? Please keep it as 2.21.0 for now

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
...
+## [2.23.0] - 2022-11-21
+
+### Fixed
+-   Updated the TUTORIAL.md documentation to include link to video, corrected
+    Redirect URI, and gave additional information on potential error.
+    
+## [2.22.0] - 2022-11-21
+
+### Added
+-   Updated the documentation to include a Code of Conduct
```
Comment (stephanebruckert) [inline CHANGELOG.md:18]: Don't create new versions, please add the changes in `Unreleased`

Comment (alhamilton1111) [reply CHANGELOG.md]: updated.

File: setup.py (hunk 2 above)
Comment (alhamilton1111) [reply setup.py]: updated, thank you!

File: TUTORIAL.md (hunk 1 above)
Comment (alhamilton1111) [reply TUTORIAL.md]: updated. Thanks!

**Pull request closed without merging**